CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ### Fixes
 - When a jinja value is undefined, give a helpful error instead of failing with cryptic "cannot pickle ParserMacroCapture" errors ([#2110](https://github.com/fishtown-analytics/dbt/issues/2110), [#2184](https://github.com/fishtown-analytics/dbt/pull/2184))
 
+### Features
+- Support for appending query comments to SQL queries. ([#2199](https://github.com/fishtown-analytics/dbt/pull/2199))
```

Comment:
Oops, sorry, one last thing! Can you add the issue to this changelog entry as well? ` [#2138](https://github.com/fishtown-analytics/dbt/issues/2138)` should be it